mandrill-api.gemspec

```diff
@@ -1,12 +1,12 @@
 Gem::Specification.new do |s|
     s.name = 'mandrill-api'
-    s.version = '1.0.54'
+    s.version = '1.0.55'
     s.summary = 'A Ruby API library for the Mandrill email as a service platform.'
-    s.description = s.summary
-    s.authors = ['Mandrill Devs']
-    s.email = 'community@mandrill.com'
+    s.description = s.summary + "\noriginal repository: https://bitbucket.org/mailchimp/mandrill-api-ruby"
+    s.authors = ['Mandrill Devs', 'current verstion: Shippio\'s dev']
+    s.email = 'it@shippio.com'
```

Comment:
Didn't know this was a thing :p
TIL again

Reply:
I didn't remember it until I saw it in the group management :D